keyboards/phantagom/baragon/info.json

```diff
@@ -0,0 +1,72 @@
+{
+    "manufacturer": "Dennis Kruyt",
+    "keyboard_name": "phantagom/baragon",
+    "maintainer": "dkruyt",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "encoder": true,
+        "encoder_map": true
```

Comment:
it wil not work or cannot? I tested and it is working, or should this not be in info.json? Stil working out what should be in info.json.

Reply:
It will not work not work under all circumstances, ie. if a user chooses not to use encoder map feature and omits the necessary code from their keymap, compilation will fail with it being enabled at the keyboard level. `"encoder_map": true` should be removed from the `info.json` file.